tests/unit/conftest.py

```diff
@@ -16,9 +16,12 @@
 
 import datetime
 from typing import Any
+from unittest import mock
 from unittest.mock import patch
 
+import craft_store
 import pytest
+from craft_store.models.registered_name_model import RegisteredNameModel
```

Comment:
Do we want to alias them here as well ?

Reply:
Probably not actually alias it, but I did change it to pull from the new namespace so we get the alias for free.